plugins/kosync.koplugin/main.lua

```diff
@@ -910,6 +910,9 @@ end
 
 function KOSync:onKOSyncToggleAutoSync(toggle, from_menu)
     if toggle == self.settings.auto_sync then return end
+    if top_widget:handleEvent(event) then
+        return
+    end
```

Comment:
No, I mean we just need `return true`.
By giving a reference to UIManager I tried to explain why, sorry if I wasn't clear.